enhancements/microshift/audit-log-configuration-options.md

```diff
@@ -0,0 +1,179 @@
+# Configurable Audit Logging for MicroShift
```

Comment:
You will need to retain the metadata section from the top of the template in that format in order for the linter to be happy.

The linter also enforces that the document includes all of the required sections (see the list of errors about the missing titles). If a section does not apply, you can just put "N/A" in the text.